packages/slice-machine/server/src/api/slices/create/index.ts

```diff
@@ -35,6 +35,7 @@ const copyTemplate = async (
     await copy(templatePath, path.join(env.cwd, from, sliceName), {
       componentName: sliceName,
       componentId: snakelize(sliceName),
+      pascalId: pascalize(sliceName),
```

Comment:
what's this for?

Reply:
wait does that mean if it's rename we should change that part as well ?

Reply:
Types are generated after the ID, and pascalcased by convention.

Just like the component name in the export, we can't really do much about it. At worst, it'll break the time for Prismic JavaScript users.